ember-file-upload/src/components/file-dropzone.ts

```diff
@@ -219,9 +223,15 @@ export default class FileDropzoneComponent extends Component<FileDropzoneSignatu
     // }
 
     if (this.dataTransferWrapper) {
-      const addedFiles = this.addFiles(this.files);
-      this.args.onDrop?.(addedFiles, this.dataTransferWrapper);
+      let files;
+      if (this.allowFolderDrop) {
```

Comment:
We can avoid defining a getter for this property because `undefined` is falsy anyway

```suggestion
      if (this.args.allowFolderDrop) {
```